docs/learn/getting-started/editors.md

```diff
@@ -50,11 +50,11 @@ Make a `workspace.code-workspace` in your project folder with the name of your g
 Set Alire to look for a workspace.code-workspace in whatever directory you're
 trying to open.
 
-`alr config --set editor.cmd "code workspace.code-workspace"`
-
-On Mac this might look something more like:
-
-`alr config --set editor.cmd "/Applications/VisualStudioCode.app/Contents/Resources/app/bin/code workspace.code-workspace"`
+<Tabs groupId="operating-systems">
```

Comment:
Sorry, this is because I cannot test locally. I tried again to set up a yarn environment of the required version, and I've given up again. I'll update blindly again, let's hope these changes are enough.

Reply:
OK, I can now build locally and I can reproduce the error in the GitHub check, but I don't know how to fix it. The problem is that ${SOMETHING} inside the <Tabs> node is evaluated, and I haven't found a way to escape the `$`

Reply:
I manage to make it work on a docker and the error is gone. I could not see the result though. I exposed the 3000 port and then connected locally to it, nothing responded... I hope it is correctly displayed.

Reply:
Are you on some Linux distro? Just install yarn v1.22. Inside the repo it will automatically use the version that is found in the repo in .yarn/releases/.

Reply:
Yes, on Ubuntu 22.04. I tried installing in /usr/local, because I didn't know at that time it is provided in Ubuntu as yarnpkg. But now, I've uninstall from /usr/local and used the yarnpkg Ubuntu package, but it also fails:
```
$ yarnpkg start
file:///home/mgr/src/github/mgrojo/ada-lang-io/.yarn/__virtual__/@docusaurus-core-virtual-7264c45998/0/cache/@docusaurus-core-npm-2.2.0-7ed3f550ad-ff47e6cf85.zip/node_modules/@docusaurus/core/bin/docusaurus.mjs:27
await beforeCli();
^^^^^

SyntaxError: Unexpected reserved word
    at Loader.moduleStrategy (internal/modules/esm/translators.js:133:18)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
    at async link (internal/modules/esm/module_job.js:42:21)
```

In the Node docker, it was straightforward.